_docs/_latest/setup/index.md

```diff
@@ -18,6 +18,11 @@ Install Forseti Security to help protect your GCP environments by monitoring you
 ensure that role-based access controls are set as you intended. The setup wizard automatically
 determines setup information, generates a deployment template, and creates a Forseti deployment.
 
+**[Migrating Forseti Security]({% link _docs/latest/setup/migrate.md %})**
+
+Migrate to Terraform an existing Forseti Security deployment created
```

Comment:
(minor) Update to `Migrate to Terraform from an existing...`

Reply:
Fixed in a6f379013f9414dffd0d49cf18f5d5e906d0d8af.